dask_distance/_utils.py

```diff
@@ -0,0 +1,31 @@
+import dask
+import dask.array
+
+import dask_distance._compat
+
+
+def _bool_cmp_mtx_cnt(u, v):
```

Comment:
Might rename this down the road. Just picked a name that made sense for now. As it is internal, no one should care that we rename it. Still leaving the note here anyways.